rag/build_vector_db.py

```
@@ -0,0 +1,130 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[4]:
+
+
+get_ipython().system('pip install -r ../requirements.txt')
```

Ruff (0.13.1)

Undefined name `get_ipython`

(F821)